### ISSUE TITLE
Fix S109: Rule should not raise FP when using enums

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-csharp#1244]
### Description

From [Google Group thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/sonarqube/LwDFQpA1X-w/Ut7KLM6GCgAJ).

### Repro steps

```
public enum AppScreen
{
   Main = 1,
   Faq = 2, // FP
   Contact = 3 // FP
}
```
